IdentityCore/src/requests/broker/MSIDSSOExtensionSignoutRequest.m

```diff
@@ -143,6 +143,7 @@ - (void)executeRequestWithCompletion:(nonnull MSIDSignoutRequestCompletionBlock)
     signoutRequest.clearSSOExtensionCookies = self.clearSSOExtensionCookies;
     signoutRequest.wipeAccount = self.shouldWipeAccount;
     signoutRequest.wipeCacheForAllAccounts = self.shouldWipeCacheForAllAccounts;
+    signoutRequest.clientBrokerKeyCapabilityNotSupported = self.requestParameters.clientBrokerKeyCapabilityNotSupported;
```

Comment:
What does this do?

Reply:
This is to not use broker key for unsigned runtime app. Like other broker request. Signout request missed this mapping